R/FilterState.R

```diff
@@ -364,13 +364,16 @@ FilterState <- R6::R6Class( # nolint
     format = function(indent = 0) {
       checkmate::assert_number(indent, finite = TRUE, lower = 0)
 
-      sprintf(
-        "%sFiltering on: %s\n%1$s  Selected values: %s\n%1$s  Include missing values: %s",
-        format("", width = indent),
-        self$get_varname(deparse = TRUE),
-        paste0(format(self$get_selected(), nsmall = 3), collapse = " "),
-        format(self$get_keep_na())
-      )
+      # List all selected values separated by commas.
+      values <- paste(format(self$get_selected(), nsmall = 3, justify = "none"), collapse = ", ")
+      paste(c(
+        strwrap(sprintf("Filtering on: %s", self$get_varname(deparse = TRUE)), indent = indent),
+        # Add wrapping and progressive indent to values enumeration as it is likely to be long.
+        strwrap(sprintf("Selected values: %s", values),
+          width = 76, indent = indent + 2, exdent = indent + 4
```

Comment:
I think the width should be the additional argument of the method

Reply:
Shall we set limits on this argument, e.g. 30-120?

Reply:
we can checkmate::assert_int(width, lower = 30L, upper = 120L) the arg to be in the raised ranged.